.github/workflows/language-reference.yaml

```diff
@@ -30,22 +30,14 @@ jobs:
           fetch-depth: 0
           ssh-key: ${{ secrets.DOCS_KEY }}
 
-      - name: Cache Coursier
-        uses: actions/cache@v1
-        with:
-          path: ~/.cache/coursier
-          key: sbt-coursier-cache
-
-      - name: Cache SBT
-        uses: actions/cache@v1
-        with:
-          path: ~/.sbt
-          key: sbt-${{ hashFiles('**/build.sbt') }}
+      - name: Cache coursier and sbt
+        uses: coursier/cache-action@v6
```

Comment:
Note that `setup-java` does support [sbt caching](https://github.com/actions/setup-java#caching-sbt-dependencies). Instead of using `coursier/cache-action`, you could set the `cache: sbt` input parameter of `actions/setup-java`.

Reply:
ooooo, I didn't know that! Thanks! I'll make this change.